static/lib/systemd/system/snapd.sshd-keygen.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Generate sshd host keys
+Before=sshd.service
```

Comment:
I would much rather see an either:
 - first boot only service
 - if-not-exists service